fast64_internal/utility.py

```diff
@@ -137,33 +183,56 @@ def restoreHiddenState(hiddenState):
         layerCol.hide_viewport = True
 
 
-def readFile(filepath):
+def readFile(filepath: str | Path) -> str:
```

Comment:
How about [os.PathLike](https://docs.python.org/3/library/os.html#os.PathLike)
File descriptors should technically also work with this function but fast64 never uses them so meh